authors/apps/articles/pagination.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class CustomPagination(PageNumberPagination):
+    page_size = 10
+    page_size_query_param = 'page_size'
+    max_page_size = 100
```

Comment:
Can you explain `page size` and `max_page_size`? Add brief explanation as a comment on the class. I also see a PAGE_SIZE=100 in settings, where does that fall into?

Reply:
Okay